migrations/env.py

```diff
@@ -53,6 +53,7 @@ def run_migrations_offline():
     context.configure(
         url=url,
         render_as_batch=True,
+        compare_type=True,
```

Comment:
I checked and `compare_type` is actually already set to be true here:

https://github.com/WildMeOrg/houston/blob/ea1a1ca63b9b23d77dc131e3e79e9fab1a78f2f3/app/extensions/flask_sqlalchemy/__init__.py#L82-L84

So I don't think this line actually does anything

(I checked the value of `self._user_compare_type` in `alembic/runtime/migration.py`)

Reply:
Confirmed. I reran the migration with tthis change and the migration file is identical.